lib/sensors/i2c_sensors.hpp

```diff
@@ -6,17 +6,21 @@
 
 namespace hyped::sensors {
 
+enum class I2cConfigureResult { kFailure = 0, kSuccess };
```

Comment:
Do we need a separate enum class for each protocol or could we just define a generic sensor configure result in types.hpp or in a new io/consts.hpp or similar? Surely we would still know what fails since we will be logging it?

Reply:
See below